eng/pipelines/templates/steps/detect-api-changes.yml

```diff
@@ -0,0 +1,25 @@
+parameters:
+  ArtifactPath: $(Build.ArtifactStagingDirectory)
+  Artifacts: []
+
+steps:
+  - pwsh: |
```

Comment:
Why does this exist? Why not just set this as a normal static variable?

Reply:
Not required. Removed this file.